# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openssl/hotfix-mips4.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- openssl-3.2.1/Configure.vanilla	2024-06-20 21:13:37.476433856 +0200
+++ openssl-3.2.1/Configure	2024-06-20 21:13:51.857433021 +0200
@@ -1490,7 +1490,7 @@
         # minimally required architecture flags for assembly modules
         my $value;
         $value = '-mips2' if ($target =~ /mips32/);
-        $value = '-mips3' if ($target =~ /mips64/);
+        $value = '-mips4' if ($target =~ /mips64/);
         unshift @{$config{cflags}}, $value;
         unshift @{$config{cxxflags}}, $value if $config{CXX};
 }
